translations/README/KOREAN.md

```diff
@@ -1,281 +1,357 @@
+이 튜토리얼은 [다른 언어](/translations/README.md)로도 제공됩니다. 
 
+|     [Arabic](/translations/README/ARABIC.md)     |  [Bangla](/translations/README/BANGLA.md)  | [Chinese (Traditional)](/translations/README/CHINESE_TRADITIONAL.md) |            [English](/README.md)             |  [French](/translations/README/FRENCH.md)  
+| :---------------------------------------------: | :---------------------------------------: | :-----------------------------------------: | :---------------------------------------: | :---------------------------------------: |
+|  [German](/translations/README/GERMAN.md)  |      [Hindi](/translations/README/HINDI.md)      | [Italian](/translations/README/ITALIAN.md) | [Japanese](/translations/README/JAPANESE.md) |  [Korean](/translations/README/KOREAN.md)  |
+  [Polish](/translations/README/POLISH.md)  | [Portuguese](/translations/README/PORTUGUESE.md) | [Russian](/translations/README/RUSSIAN.md) |  [Serbian](/translations/README/SERBIAN.md)  | [Spanish](/translations/README/SPANISH.md) |
+   [Turkish](/translations/README/TURKISH.md) | [Ukrainian](/translations/README/UKRAINIAN.md) |
 
-## 준비
+> 프로젝트 문서 번역을 하고 싶다면 환영입니다. [`번역 가이드`](/translations/README.md)를 읽고 기여해주세요.
 
-작업을 위해서 몇 가지 준비를 해보겠습니다.
+---
 
-1. GitHub 계정에 로그인하세요. 만약 아직 계정이 없으시다면 [GitHub에 가입](https://github.com/join)하세요. 작업 진행을 위해 [GitHub Hello World 튜토리얼](https://guides.github.com/activities/hello-world/)을 정독하시는 것을 추천드립니다.
-1. [GitHub 데스크탑 앱](https://desktop.github.com/)을 설치하세요.
-   - 커맨드 라인에서 Git을 사용하는 것이 편하다면 GitHub 데스크탑 앱을 설치하지 않으셔도 됩니다.(커맨드 라인에서 Git 사용에 대한 가이드는 이 프로젝트와 유사한 프로젝트인 [first-contributions](https://github.com/Syknapse/first-contributions)을 참고하세요.) 또는
-   - [VS Code](https://code.visualstudio.com/)를 사용하는 경우 통합 Git이 제공되므로 에디터에서 직접 필요한 작업을 수행할 수 있습니다.
+### 준비! :)
 
-이제 모든 준비가 완료되었으므로 본격적으로 프로젝트 기여를 위한 작업을 시작해보겠습니다.
+참고: 이 튜토리얼은 GitHub PC를 기반으로 합니다. [터미널을 편하게 사용하신다면 해당 튜토리얼로 이동하세요 (여기를 클릭하세요)](/terminal_tutorial.md)
 
----
+먼저 작업 환경을 설정해봅시다.
+
+1. GitHub 계정에 로그인하세요. 만약 아직 계정이 없으시다면 [GitHub에 가입하세요](https://github.com/join). 원활한 작업 진행을 위해 [GitHub Hello World 튜토리얼](https://guides.github.com/activities/hello-world/)을 읽고 시작하는 것을 추천드립니다.
+2. [GitHub 데스크탑 앱](https://desktop.github.com/)을 다운로드 후 설치하세요.
+   - 커맨드 라인에서 Git을 사용하는 것이 편하다면 GitHub 데스크탑 앱을 설치하지 않으셔도됩니다. [커맨드 라인(CLI) 튜토리얼 링크](/terminal-tutorial.md).
```

Comment:
this link should forward to korean terminal tutorial.

Reply:
Updated the link